File: Sheet #1 (Data type - Conditions)/C++/K-Max and Min.cpp
#include <iostream>
using namespace std;

long long max_num(long long a, long long b, long long c)
{
    long long max;

    if (a >= b && a >= c)
        max = a;

    if (b >= a && b >= c)
        max = b;

    if (c >= a && c >= b)
        max = c;

    if (a == b && a == c)
        max = a;

    return (max);
}

long long min_num(long long a, long long b, long long c)
{
    long long min;

    if (a <= b && a <= c)
        min = a;

    if (b <= a && b <= c)
        min = b;

    if (c <= a && c <= b)
        min = c;

    if (a == b && a == c)
        min = a;

    return (min);
}

int main()
{
    long long a, b, c, max;

    cin >> a >> b >> c;

    cout << min_num(a, b, c) << ' ' << max_num(a, b, c) << endl;

    return (0);
}

File: Sheet #1 (Data type - Conditions)/python/K-Max and Min.py
#!/usr/bin/python3
'''
Given 3 numbers A, B and C,
Print the minimum and the maximum numbers.
'''

user_input = input("")

string_values = user_input.split()
values = [float(x) for x in string_values]

max_num = max(values)
min_num = min(values)

print(f"{int(min_num)} {int(max_num)}")
